pkgs/development/python-modules/grep-ast/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+
+  pathspec,
+  tree-sitter-languages,
+}:
+
+buildPythonPackage rec {
+  pname = "grep-ast";
+  version = "0.3.3";
+
```

Comment:
```suggestion
  version = "0.3.3";
  pyproject = true;

```

Reply:
It isn't a pyproject-based library.

Reply:
Yes, but it should work nonetheless. It's counter intuitive I agree.

Reply:
It doesn't, though. I get this error.

<details>
```
greg@jude ~/s/nixpkgs init-grep-ast @ nix build ".#python3Packages.grep-ast"
warning: Git tree '/home/greg/src/nixpkgs' is dirty
error: builder for '/nix/store/0ybl3jji0zvn2pkxmyz6mis7c4bdw6xv-python3.12-grep-ast-0.3.3.drv' failed with exit code 1;
       last 10 log lines:
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
       > ModuleNotFoundError: No module named 'setuptools'
       >
       > ERROR Backend 'setuptools.build_meta:__legacy__' is not available.
       For full logs, run 'nix log /nix/store/0ybl3jji0zvn2pkxmyz6mis7c4bdw6xv-python3.12-grep-ast-0.3.3.drv'.
```
</details>

Reply:
Yes, add `build-system = [ setuptools ];`

Reply:
Done